components/forms/BookDemo.tsx

```diff
@@ -0,0 +1,115 @@
+                </div>
+              </Link>
+            </div>
+          </div>
+          {/* <div className="flex justify-center w-full items-center h-full">
+            <div className="w-full max-w-sm h-full">
+              <Link
+                href="#"
+                className="flex flex-col md:flex-row lg:flex-row w-full h-full items-center justify-between rounded-lg border border-input bg-background p-4 shadow transition transform duration-200 hover:scale-105 hover:bg-muted focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring text-blue-800 hover:text-blue-700"
+                prefetch={false}
+              >
+                <img
+                  src={JeanCircle.src}
+                  alt="Europe Portrait"
+                  className="hidden md:block w-18 h-18 rounded-full lg:mr-4 md:mr-4"
+                />
+                <div className="flex-grow text-center md:text-left lg:text-left">
+                  <div className="font-medium text-lg">Europe</div>
+                  <div className="text-muted-foreground text-xs">
+                    Meet with Jean (English and French)
+                  </div>
+                </div>
+                <div className="flex-shrink-0">
+                  <ChevronRightIcon className="h-6 w-6 text-muted-foreground" />
+                </div>
+              </Link>
+            </div>
+          </div> */}
+          <div className="flex justify-center w-full items-center h-full">
+            <div className="w-full max-w-sm h-full">
+              <Link
+                href="https://meetings-eu1.hubspot.com/levi-jackson?uuid=2eb9cf82-25fe-40b6-86c3-466c365d34a5"
+                className="flex flex-col md:flex-row lg:flex-row w-full h-full items-center justify-between rounded-lg border border-input bg-background p-4 shadow transition transform duration-200 hover:scale-105 hover:bg-muted focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring text-blue-800 hover:text-blue-700"
+                prefetch={false}
+              >
+                <img
+                  src={LeviCircle.src}
+                  alt="Oceania and Asia Portrait"
+                  className="hidden md:block w-18 h-18 rounded-full lg:mr-4 md:mr-4"
+                />
+                <div className="flex-grow text-center md:text-left lg:text-left">
+                  <div className="font-medium text-lg">Oceania and Asia</div>
+                  <div className="text-muted-foreground text-xs">
+                    Meet with Levi (English)
+                  </div>
+                </div>
+                <div className="flex-shrink-0">
+                  <ChevronRightIcon className="h-6 w-6 text-muted-foreground" />
+                </div>
+              </Link>
+            </div>
+          </div>
+        </div>
+      </div>
+    </>
+  );
+};
+
+export default DemoForm;
+
+function ChevronRightIcon(props) {
```

Comment:
Believe this uses `react/icons` can we use the icon from there instead

Reply:
✅